cluster-scope/overlays/nerc-ocp-infra/clustersecretstores/nerc-cluster-secrets_patch.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ClusterSecretStore
+metadata:
+  name: nerc-cluster-secrets
+spec:
+  provider:
```

Comment:
Oops I guess I didn't submit this comment yesterday. Is this configuration set up internally in Vault? If not I would be happy to get that started